.github/renovate.json

```diff
@@ -70,6 +70,11 @@
       "matchPackagePrefixes": ["org.jacoco"],
       "allowedVersions": "!/0.8.9/"
     },
+    {
+      "matchManagers": ["maven"],
```

Comment:
Please add a description (text + e.g. ticket link) like many of the rules have :)

Reply:
I thought about adding a comment but remembered JSON has no comments, I didn't know about the "description" field (but should've since it's used elsewhere :see_no_evil:)